docs/pages/includes/configure-event-handler.mdx

```diff
@@ -57,7 +57,7 @@ Teleport event handler 0.0.1 07617b0ad0829db043fe779faf1669defdc8d84e
 
 Follow-along with our getting started guide:
 
-https://goteleport.com/setup/guides/fluentd
+https://goteleport.com/docs/management/export-audit-events/fluentd/
```

Comment:
Any reason not to use a relative path here?

Reply:
I missed that one. We'll fix it in a subsequent PR